metricflow/model/transformations/dbt_to_metricflow.py

```diff
@@ -235,11 +238,11 @@ def build_proxy_metric(self, dbt_metric: DbtMetric) -> Metric:
         if dbt_metric.calculation_method == "derived":
             raise RuntimeError("Cannot build a MetricFlow proxy metric for `derived` DbtMetric")
 
-        where_clause_constraint: Optional[WhereClauseConstraint] = None
+        where_filter: Optional[WhereFilter] = None
         if dbt_metric.filters:
-            where_clause_constraint = WhereClauseConstraint(
-                where=self.build_where_stmt_from_filters(filters=dbt_metric.filters),
-                linkable_names=[filter.field for filter in dbt_metric.filters],
+            # TODO: Figure out how to migrate.
```

Comment:
I think all of this code will go away since we're not supporting this converter in our next release.

Reply:
Chatted with @QMalcolm earlier and confirmed that they going away. Also the reason for the skipped tests in a later commit.